client/client.go

```diff
@@ -191,22 +191,32 @@ func loadServerTrust(conf Config) (*x509.CertPool, error) {
 		return nil, nil
 	}
 
-	certs, err := x509.SystemCertPool()
+	syscerts, err := x509.SystemCertPool()
 	if err != nil {
 		return nil, err
 	}
 
 	// Read certificate file.
-	cacert, err := ioutil.ReadFile(conf.ServerCert)
+	servcert, err := ioutil.ReadFile(conf.ServerCert)
 	if err != nil {
 		return nil, err
 	}
-	certs.AppendCertsFromPEM(cacert)
 
-	if len(certs.Subjects()) == 0 {
+	if len(servcert) == 0 {
```

Comment:
yup, `len()` is smart enough to return 0 if `severcert == []byte{}` or `servercert == nil`